docs/source/dataframe-create.rst

```diff
@@ -57,23 +59,61 @@ Pandas:
 Creating
 --------
 
-Reading from various locations
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Read from CSV
+~~~~~~~~~~~~~
 
-For text, CSV, and Apache Parquet formats, data can come from local disk,
-the Hadoop File System, S3FS, or other sources, by prepending the filenames with
-a protocol:
+You can use :func:`read_csv` to read one or more CSV files into a Dask DataFrame.
+It supports loading multiple files at once:
+  
+.. code-block:: python
+
+   >>> df = dd.read_csv('myfiles.*.csv')
```

Comment:
Naive user hat on... It may not be clear that the `*` in the filename string indicates that Dask should load multiple files. Perhaps there's some context here that's missing in the diff view 😄 If not, can the mechanism by which Dask knows that this should result in multiple files be explained briefly?